doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex

```diff
@@ -25,7 +25,7 @@ \subsection{HAMT Nomenclature}\label{hamt-nomenclature}
 C & J/kgC & Specific heat capacity of dry material \tabularnewline
 $C_i^h$ & J/C & Heat Capacitance of cell i \tabularnewline
 $C_i^w$ & kg & Moisture Capacitance of cell i \tabularnewline
-$c_{w}$ & J/KgC & Specific heat capacity of water (= 4,180~J/kg$^{o}$C@ 20$^{o}$C) \tabularnewline
+$c^{w}$ & J/KgC & Specific heat capacity of water (= 4,180~J/kg$^{\circ}$C@ 20$^{\circ}$C) \tabularnewline
```

Comment:
Is this right?  It's c^w?  :shrug:

Reply:
That's what's used in the equation. We can change either here to c^w or a few place in the equations to c_w.

Reply:
:shrug: OK, thanks, we'll go with it.